libs/core/src/lib/action-sheet/action-sheet.component.ts

```diff
@@ -53,7 +53,7 @@ export class ActionSheetComponent implements AfterContentInit, AfterViewInit, On
         this._initializeChildrenState();
         this._setItemsProperties();
     }
-    private _compact = false;
+    private _compact: boolean = undefined;
```

Comment:
It's initialized to undefined by default.

Reply:
Maybe we also should have a getter for the `compact` property for different purposes when the customer will work with the component programmatically?

Reply:
it is initialized to undefined because the content density service logic is working that way.[ In these lines](https://github.com/SAP/fundamental-ngx/pull/5552/files/38089fdb1394bd59d8423e3d640ce75be8859a8e#diff-0eeca74488ee851d7cf8c80beaf7f7866f1e618f965eb8846eabc89d9a44a69eR146), there is a check whether user has defined a value for `compact`, if not, it will register to the content density service. 
Having it set to `false` by default would mean it will not register to the content density service even if user has not specified a `compact` value.

Reply:
I fully agree with you but `private _compact: boolean` also will initialize `_compact` with `undefined` value.

Reply:
ah, yes, i had it like that before but then felt i should initialize it to something. Will revert the `undefined` :)

Reply:
have made the changes, please check. @platon-rov